probcal/enums.py

```diff
@@ -13,6 +13,7 @@ class HeadType(Enum):
     POISSON = "poisson"
     DOUBLE_POISSON = "double_poisson"
     NEGATIVE_BINOMIAL = "negative_binomial"
+    MULTI_CLASS = "multi_class"
```

Comment:
We can probably drop this `HeadType` since we're not training any `MultiClassNN` models